com.unity.uiwidgets/Runtime/engine/UIWidgetsMessageManager.cs

```diff
@@ -74,18 +74,28 @@ public class UIWidgetsMessageManager: MonoBehaviour {
             }  
         }
 
+        Queue<string> messages = new Queue<string>();
+
         void OnUIWidgetsMethodMessage(string message) {
-            JSONObject jsonObject = (JSONObject)JSON.Parse(message);
-            string channel = jsonObject["channel"].Value;
-            string method = jsonObject["method"].Value;
-            var args = new List<JSONNode>(jsonObject["args"].AsArray.Children);
-            if (string.IsNullOrEmpty(channel) || string.IsNullOrEmpty(method)) {
-                Debug.LogError("invalid uiwidgets method message");
-            }
-            else {
-                MethodChannelMessageDelegate exists;
-                _methodChannelMessageDelegates.TryGetValue(channel, out exists);
-                exists?.Invoke(method, args);
+            messages.Enqueue(message);
+        }
+
+        public void handlePlatformMessage(){
```

Comment:
we should wrap this function with #define since it is only required for Android by now